### ISSUE TITLE
button disabled state missing

### ISSUE THREAD

**Issue by jnarowski** [creativetimofficial/vue-light-bootstrap-dashboard-pro#1]
At least for primary button but maybe others, the CSS class for disabled isn't there and the button is totally white when using disabled="disabled"

**Comment by cristijora** [comment]
@jnarowski Thanks a lot for pointing out. Will fix it asap.

**Comment by cristijora** [comment]
Planning a release this week.
Until then, you could replace lines 28,29 in `mixins/_buttons.scss`
with this code
```css
 border-color: $btn-color;
 background-color: $btn-color;
 cursor: not-allowed;
```

**Comment by cristijora** [comment]
Fixed in 1.1.0 release. See [Changelog](https://github.com/creativetimofficial/vue-light-bootstrap-dashboard-pro/blob/master/CHANGELOG.md) and [migration guide](http://vuejs.creative-tim.com/vue-light-bootstrap-dashboard-pro/documentation/#/migration_guide). To get the new version go to [Downloads section](https://www.creative-tim.com/downloads) and redownload the product.
